cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorState.java

```diff
@@ -141,6 +146,14 @@ protected boolean removeEldestEntry(Map.Entry<String, AnomalyState> eldest) {
         Timer timer = dropwizardMetricRegistry.timer(
             MetricRegistry.name(ANOMALY_DETECTOR_SENSOR, String.format("%s-detect-to-fix-complete-timer", anomalyType.toString().toLowerCase())));
         _anomalyDetectToFixCompleteTimer.put(anomalyType, timer);
+
+        Counter counter = dropwizardMetricRegistry.counter(
```

Comment:
If these `counter` and `counter1` variables are not actually used anywhere other than the `map.put` call, can you just in-line them? 

Same with `timer` variable above?